awsmp/_driver.py

```diff
@@ -102,19 +104,19 @@ def get_client(service_name="marketplace-catalog", region_name="us-east-1"):
     return boto3.client(service_name, region_name=region_name)
 
 
-def get_response(changeset_stringified: ChangeSetType, changeset_name: str) -> ChangeSetReturnType:
+def get_response(changeset: List[ChangeSetType], changeset_name: str) -> ChangeSetReturnType:
     """
     Request to AWS and get response of either success of failure
 
-    :param ChangeSetType changeset_stringified: string type of changeset
+    :param ChangeSetType changeset: changeset dictionary
```

Comment:
These types/names seem inconsistent. It's called `changeset` implying a single changeset, expects a list of changesets, but we document it a single `ChangeSetType`, labeled as a dictionary